### PR TITLE
Added 4 pages

### PR DESCRIPTION
**Pull request by MacaLaur**
Created html pages for homepage, dog listing, blog and checkout page for Adopt a Dog site.

### CODE REVIEW

**Comment by MacaLaur** [comment]
Add css styling to site

File: blog.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        <img src="images/cart-icon.svg" alt="white icon outline of a shopping cart" width="40">
+        <button>Checkout</button>
+      </div>
+      <img class="logo" src="images/adoptadog-logo.svg" alt="adopt a dog logo of a white dog silhouette sitting with the adopt a dog name beside it" width="300">
+      <h1><em>Seattle's premiere dog adoption agency</em></h1>
+    </header>
+    <nav class="top-nav">
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li>/</li>
```
Comment (jimmynono) [inline blog.html:22]: There are better ways to handle these / marks. The advanced (best) way would be to use a css pseudo element. You could research that (you'll need to learn how to use the after selector and content) or you could just add a slash after the Home and Dogs anchor tags

**Comment by MacaLaur** [comment]
update css

File: styles/stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+  padding: 0;
+}
+
+header {
+  background-image: url("../images/header-image.jpg");
+  padding: 7em;
+  color: #55C632;
+}
+
+.float-right {
```
Comment (jimmynono) [inline styles/stylesheet.css:24]: I'd love for you to find another way to layout with out using floats. Consider using flexbox, grid, or position to move these boxes around. Float is just a pain to work with and we want to minimize its use as we have newer tools to use.

File: styles/stylesheet.css [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+}
+
+.home-intro img, .blog-post img {
+  float: left;
+  width: 30%;
+}
+
+.home-intro p, .home-intro h1, .blog-post p, .blog-post h2{
+  margin: 1em 2em;
+  display: table;
```
Comment (jimmynono) [inline styles/stylesheet.css:68]: Don't use display table for this kind of thing. We only want to use display table when we are displaying tables of data

File: styles/stylesheet.css [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+  margin-top: 2em;
+}
+
+.flexbox-container {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  align-content: center;
+  align-items: flex-start;
+  clear: both;
```
Comment (jimmynono) [inline styles/stylesheet.css]: You shouldn't need this clear

File: index.html [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+      </article>
+      <article class="flex-dog">
+        <img src="images/poppy-card.jpg" alt="white bichon shih tzu mix running" width="175" onclick="alert('Poppy, Shih Tzu Mix, $123.45')">
+        <h2>Poppy</h2>
+        <h3><b>Cost to Adopt:</b> $123.45</h3>
+        <p>90s ipsum Bill gates world wide web smarter child rollerblades nylon windbreaker, leggings aviators I will always love you wayne gretzky clueless tupac shakur.</p>
+        <button onclick="addToTotal(123.45)">Adopt</button>
+      </article>
+      <article class="flex-dog">
+        <img src="images/jack-card.jpg" alt="white and light brown beagle" width="175" onclick="alert('Jack, Beagle, $123.45')">
```
Comment (jimmynono) [inline index.html:71]: One way you could improve this, is to create a function that calls a function that lives in your JS file, and then that JS function calls the alert. It's a bit overkill for something this simple, but good practice.

File: script/dogs.js [hunk 6]
```diff
@@ -0,0 +1,3 @@
+$('.flex-dog').hover(function(e) {
+  $('.flex-dog').addClass('dog-border');
```
Comment (jimmynono) [inline script/dogs.js:2]: This is close, but would be nice if this file was on the index page as well. I had to dig to figure out why nothing was happening and found this was only on the dogs page.

Also, see if you can find a way to add a class (or even better toggle the class) on an individual tile.

File: script/checkout.js [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    zipCode: zipCode.value,
+    adoptStatus: adoptStatus.value,
+    location: location.value
+  });
+});*/
+
+$('form').submit(function(e) {
+  e.preventDefault();
+  alert('Thank you. The form information has been received.');
+  console.log($('form').serialize());
```
Comment (jimmynono) [inline script/checkout.js:28]: So nice, isn't it?

**Comment by MacaLaur** [comment]
Created a readme.md and added testing.txt file to explain tests, user info, analytics for site.

**Pull request closed without merging**